assets/styles/informed.scss

```diff
@@ -2032,3 +2032,45 @@ govuk-checkboxes__item {
 .govuk-error-message {
   font-size: 19px;
 }
+
+.govuk-table__cell {
+  font-size: 19px;
+
+  @include media(mobile) {
+    font-size: 16px;
+  }
+}
+
+caption {
+  text-align: left;
+}
+
+.caption-xlarge {
+  font-size: 27px;
+  color: #505a5f;
+
+  @include media(mobile) {
+    font-size: 18px;
+  }
+}
+
+.no-table-border {
+  .govuk-table__header,
+  .govuk-table__cell,
+  .govuk-table__row {
+    border-bottom: none;
+  }
+
+  .govuk-table__header {
+    width: 200px;
+  }
+}
+
+.no-table-header-border .govuk-table__head .govuk-table__header {
+  border-bottom: none;
+}
```

Comment:
with BEM all rules should all be the same level of specificity (no nesting), although it doesn't seem like the rest of the code is really following BEM. Essentially this could be a headache in the future if you want a table with no borders, but if we need `govuk-table__cell` to have a border  in the future (weird example, I don't know why anyone would want this..) it will be a bit of a headache to add that style since you'd have to override by adding more specificity. 

To add a modifier in the same amount of lines (I'm pretty sure this works 🤔) 
```
.govuk-table__header,
.govuk-table__cell,
.govuk-table__row {
  &--no-border {
    border-bottom: none;
  }
}
```

Reply:
Ah cool, yeah I haven't really been following BEM with all my new styles. I'll see what I can do to fix this